version.html

```diff
@@ -3,7 +3,7 @@
         <title>ACE3 Version</title>
     </head>
     <body>
-        <h1>Latest: 3.1.0.0 - <a href="https://github.com/acemod/ACE3/releases/latest">Download</a></h1>
+        <h1>Latest: 3.1.1.1 - <a href="https://github.com/acemod/ACE3/releases/latest">Download</a></h1>
```

Comment:
Check the image, "installed version" includes the build number as well, I think it's best to have them both the same format.